#book to manipulate
librarian_resources/Book book
#where to place the book
geometry_msgs/PoseStamped place_pose
#approach the placing pose from the left
bool approach_left